search/includes/classes/class-search.php

```diff
@@ -787,9 +787,7 @@ public function filter__ep_index_name( $index_name, $blog_id, $indexable ) {
 	 */
 	public function filter__ep_global_alias( $alias_name, $indexable ) {
 		// TODO: Use FILES_CLIENT_SITE_ID for now as VIP_GO_ENV_ID is not ready yet. Should replace once it is.
-		$alias_name = sprintf( 'vip-%s-%s-all', FILES_CLIENT_SITE_ID, $indexable->slug );
-
-		return $alias_name;
+		return sprintf( 'vip-%s-%s-all', FILES_CLIENT_SITE_ID, $indexable->slug );
```

Comment:
Immediately return this expression instead of assigning it to the temporary variable